config/routes.rb

```diff
@@ -44,6 +44,10 @@
     get 'plugins/:type/:id', to: 'plugins#show', as: 'plugin'
   end
 
+  resources :pages, path: 'a', as: 'member_facing_page', only: [:edit, :show] do
```

Comment:
yeah all the pages stuff still works. I left `/a/:id/edit` as a convenience so you can just append '/edit' to the url and go back to editing